clang/lib/CodeGen/Targets/AArch64.cpp

```diff
@@ -243,6 +243,7 @@ AArch64ABIInfo::convertFixedToScalableVectorType(const VectorType *VT) const {
 
     case BuiltinType::SChar:
     case BuiltinType::UChar:
+    case BuiltinType::MFloat8:
```

Comment:
Is this used by any intrinsic? Can we have test for this?

Reply:
Intrinsics are not relevant here.
This is covered by `clang/test/CodeGen/AArch64/pure-scalable-args.c`.